perf/gfoidl.Stochastics.Benchmarks/Program.cs

```diff
@@ -1,13 +1,68 @@
-﻿using BenchmarkDotNet.Running;
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Numerics;
+using System.Reflection;
 
 namespace gfoidl.Stochastics.Benchmarks
 {
     static class Program
     {
         static void Main(string[] args)
         {
-            var switcher = BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly);
-            switcher.Run(args);
+            if (args.Length != 1)
```

Comment:
Maybe an option to go to the `BenchmarkSwitcher`?

Although there' s no real win over this custom implementation.